distribution/src/main/resources/bin/elasticsearch-keystore

```diff
@@ -77,10 +77,9 @@ HOSTNAME=`hostname | cut -d. -f1`
 export HOSTNAME
 
 declare -a args=("$@")
-path_props=(-Des.path.home="$ES_HOME")
```

Comment:
This change is making it consistent with the other scripts (and another follow-up is going to make everything even more consistent).